src/Campaigns/Blocks/CampaignGoal/app/index.tsx

```diff
@@ -0,0 +1,34 @@
+import {__} from '@wordpress/i18n';
+import {Campaign} from '@givewp/campaigns/admin/components/types';
+import {getGoalDescription, getGoalFormattedValue} from '../utils';
+
+import './styles.scss';
+
+export default ({campaign}: { campaign: Campaign }) => {
+    return (
+        <div className="give-campaign-goal">
```

Comment:
The future is `givewp` in all the places `givewp-campaign-goal` 😎

Reply:
Resolved https://github.com/impress-org/givewp/pull/7702/commits/376eba218ed324e6623d439bfda1034404b20f21